loadtest/tinode.erl

```diff
@@ -27,3 +27,19 @@ get_rand_secret() ->
 shuffle(L) ->
   RandomList=[{rand:uniform(), X} || X <- L],
   [X || {_,X} <- lists:sort(RandomList)].
+
+%% Reads previously cached token for the specified user.
```

Comment:
I would add the word 'auth' somewhere to make it clearer.

Reply:
done